dockerfiles/build-extension/fpm_apk_pax_header.patch

```diff
@@ -0,0 +1,29 @@
+From 18221cceb06860395120b56cca7e270539b74073 Mon Sep 17 00:00:00 2001
```

Comment:
This file is a clone of the patch we already have in use, moved here from our docker-library repo.